docs/pipeline/cleanup/README.md

```diff
@@ -0,0 +1,27 @@
+# Cleanup SOP
+
+### Steps
+
+- Must up-to-date understanding with Pipeline Data Model
+- General strategy: preform cleanup ops before staging rerun trigger
+- Using SQL scripts to determine 
+  - affected GDS paths to be deleted
+  - affected workflows to be marked them `Failed`
```

Comment:
mention that DB clean up (`FastqListRow`) may be needed?
(noted below, but perhaps worth to point out here in those "Steps". Notes are easily overlooked.)

Reply:
It is a bit dynamic, but yah, I will pull it up.